app/authentication/session_storage.py

```diff
@@ -69,7 +78,7 @@ def _get_user_session(eq_session_id):
             logger.debug('found matching eq_session for eq_session_id in database',
                          session_id=eq_session.eq_session_id,
                          user_id=eq_session.user_id,
-                         timestamp=eq_session.timestamp.isoformat())
+                         created_at=eq_session.created_at.isoformat())
```

Comment:
Do we need to log the created time?

Reply:
We had it before (timestamp) so i've left it in. We probably don't though so worth discussing with @ajmaddaford.

Reply:
With this in it errors when loading a session if that session was started on the previous version of code